maven/impl-maven-embedded/src/test/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/pom/equipped/PomEquippedEmbeddedMavenRunningAsDaemonTestCase.java

```diff
@@ -35,7 +35,7 @@ void testDaemonShouldWaitForBuildSuccess() throws TimeoutException {
             .withWaitUntilOutputLineMatches(".*BUILD SUCCESS.*")
             .build();
 
-        Awaitility.await("Wait till thread is not be alive").atMost(20, TimeUnit.SECONDS)
+        Awaitility.await("Wait till thread is not be alive").atMost(45, TimeUnit.SECONDS)
```

Comment:
Why increasing the timeout? Did the changes in the PR resulted in longer run?

Reply:
Sounds good to me. Can I ask for a separate commit here as well?

Reply:
Yes, I will do that. thanks